float32 radius
float32 angle
float64[4] covariance
float64[2] start
float64[2] end
